src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1216,7 +1252,7 @@ public synchronized void afterDisconnect(SlaveComputer slaveComputer, TaskListen
                     if (sftpClient == null) {// system without SFTP
                         try {
                             connection.exec("rm " + fileName, listener.getLogger());
-                        } catch (Exception x) {
+                        } catch (Throwable x) {
```

Comment:
What about letting `Error` fall through. Seems like a change above and beyond findbugs